lightning/impl/base.py

```diff
@@ -52,7 +59,7 @@ def predict_proba(self, X):
             prob += 1
             prob /= 2
         else:
-            raise NotImplementedError("predict_(log_)proba only supported when"
+            raise NotImplementedError("predict_proba only supported when"
```

Comment:
I've kept NotImplementedError here as a defensive measure, but probably it is better to remove it.